### ISSUE TITLE
Suggestion : top panel to bottom

### ISSUE THREAD

**Issue by squalou** [pop-os/gnome-shell-extension-pop-cosmic#118]
Hi,

pop-shell is probably THE thing that made me come back to Gnome. So first of all : thank you for the usability-focused approach.
Now with Cosmic approach it's even better. I was fed up with adding tons of extensions to have a desktop usable ... so really thank you.

There still is one extension I add, for just one missing option : `bottom-panel`. I'd like to share why I thing bottom panel is better .. and I don't know where to discuss it so let's state that here.
Feel free to close without even commenting if it's inappropriate.

I'm a 50 / 50 keyboard / mouse user. I have 30 years of computer habits, I'm OK with changes only if they don't break basic UI rules and optimizations.

One of these golden rules are : screen borders have value, since you can move your mouse to the border without braking to reach a target. Screen borders can be seen as "infinite" areas : move 1 million pixels up, you're still on the border. Corners have even more value as you really can't miss them.
With multi screens it may be harder to justify, and to solve that, mouse can "resist" a bit when traversing to another screen.

Example:  top right corner for instance is THE place to reach for the `X` that closes a window, for mouse users.
(once upon a time, win95 users remember microsoft doing this wrong : you had to stop a few pixels before the corner to stop on the button, anyway)

For instance, when using a mouse, with a maximized browser that has "tabs on top" (which was really GREAT when chrome set that as a new standard) : tabs cannot be missed, you don't have to stop.
Same thing goes for the "traditional" applications menu bar sitting on top. 

MacOS has always been successful with a top bar ... not because the bar is at the top, but because it dynamically adapts not to be in the way. Applications menu are inside, etc.
Gnome obviously is progressing in this direction but is FAR from the goal.

Now, with a Top Bar : what happens to bottom space ? it's lost !
Sure, you can add a dock there. But still : 
- bottom left and right corners have no use.
- usual monitors have 16/9 ratio  (16/10 is best and are finally available again but still). Having Top Bar and Bottom Dock makes your screen useful space like a Cinerama 

For all these reasons I much prefer following setting
- Bottom "top bar"
- Left Dock (not extending to borders)

As a result, using a mouse or pad :
- move to right bottom corner : gets you to 'system' gnome menu (sound, network, systray-like notifications for those old apps etc).
- move to right top corner : `X` to close window
- top left corner is 'previous page' in a maximized browser for instance
- bottom left corner is 'applications' menu (if enabled)
- topmost 'infinite' area : your main app menu or browser tabs
- bottom 'infinite' area : gnome things, app menu, workspaces, notification...
- left side 'infinite' area : dock
- right side 'infinite' area : window scrollbars when there one

All this to say :
- It would be incredibly cool to enable "bottom-panel" option
- as a complement you could also have a top dock.

Thanks for reading !

And thanks once again for your amazing work.

**Comment by squalou** [comment]
apparently even Gnome guys come to their mind sometimes :)

https://www.omgubuntu.co.uk/2021/04/new-gnome-shell-mockups-explore-a-bottom-bar-layout

**Comment by Blackrucat** [comment]
This is only on gnome 40 @squalou

**Comment by squalou** [comment]
Yes sure, I was just mentioning the fact that even gnome guys were thinking about a bottom-bar as a good thing.
For 3.38 and 40, extensions do exist that do the job. But the less extensions I install, the better I am.
